src/vorta/store/models.py

```diff
@@ -90,6 +90,8 @@ class BackupProfileModel(BaseModel):
     schedule_make_up_missed = pw.BooleanField(default=True)
     validation_on = pw.BooleanField(default=True)
     validation_weeks = pw.IntegerField(default=3)
+    compaction_on = pw.BooleanField(default=True)
```

Comment:
Let's disable it by default to avoid surprising behavior.

Reply:
Sure!